src/sentry/tasks/post_process.py

```diff
@@ -115,6 +117,22 @@ def _capture_event_stats(event: Event) -> None:
     metrics.timing("events.size.data", event.size, tags=tags)
 
 
+def _update_escalating_metrics(event: Event) -> None:
+    """
+    Update metrics for escalating issues when an event is processed.
+    """
+    metrics_backend = KafkaMetricsBackend()
+    metrics_backend.counter(
+        UseCaseID.ESCALATING_ISSUES,
+        org_id=event.project.organization_id,
+        project_id=event.project.id,
+        metric_name="event_ingested",
+        value=1,
+        tags={"group": event.group_id},
```

Comment:
Is `event.group_id` a string?

Reply:
Tag keys and values must both be strings to be compatible with the Kafka schema here https://github.com/getsentry/sentry-kafka-schemas/blob/main/schemas/ingest-metrics.v1.schema.json